utils.pwsh/Setup-Target.ps1

```diff
@@ -78,6 +78,15 @@ function Find-VisualStudio {
 
     $VisualStudioData = Get-CimInstance MSFT_VSInstance
 
+    if ( $VisualStudioData.GetType() -Eq [object[]] ) {
+        $VisualStudioData = ($VisualStudioData | Sort-Object -Property Version)[0]
+    }
+
+    $VisualStudioId = "Visual Studio {0} {1}" -f @(
```

Comment:
You don't need those lines - just the 3 lines above flattening a possible array to a single object.